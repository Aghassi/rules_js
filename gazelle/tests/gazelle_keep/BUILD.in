load("@npm//@bazel/typescript:index.bzl", "ts_project")

ts_project(
    name = "gazelle_keep",
    srcs = [],
    deps = [
        "@npm//@types/jquery", # keep
    ],
)