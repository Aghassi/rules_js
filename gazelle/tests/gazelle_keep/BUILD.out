load("@npm//@bazel/typescript:index.bzl", "ts_project")

ts_project(
    name = "gazelle_keep",
    srcs = [
        "lib.ts",
        "main.ts",
    ],
    deps = [
        "@npm//@types/jquery",  # keep
    ],
)
